src/apps/frontend/contexts/auth.provider.tsx

```diff
@@ -13,19 +13,33 @@ import useAsync from './async.hook';
 
 type AuthContextType = {
   isLoginLoading: boolean;
+  isSendOTPLoading: boolean;
   isSignupLoading: boolean;
   isUserAuthenticated: () => boolean;
+  isVerifyOTPLoading: boolean;
   login: (username: string, password: string) => Promise<AccessToken>;
   loginError: AsyncError;
   loginResult: AccessToken;
   logout: () => void;
+  sendOTP: (
+    countryCode: string,
+    phoneNumber: string,
```

Comment:
We should define a `PhoneNumber` type so we don't have keep passing countryCode and `phoneNumber` (as string) separately here, and in `verifyOTP`